### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by andrewdavidmackenzie** [andrewdavidmackenzie/pigg#122] [Announcements]
This is the first release of `pigg`!

We hope there is enough functionality to give you a sense of what it is and how it could evolve, we would love to hear your feedback and ideas, via Discussions in this repo, or Issues or comments on any posts we do. Thank You.

## Initial Functionality

* Visual representation of the Raspberry Pi's 40 GPIO pins in two layouts, a "Board Pin Layout" that mimics the physical layout of the Pi's GPIO connector/header, or a "BCM Pin Layout" with only the programmable GPIO pins, ordered by BCM pin number.
* Each pin has its board pin number, name and function.
* Drop down selector to config each pin (Currently as an Input with or without pull-up/pull-down, or as an Output)
* Inputs have a visualization like an LED to show its current level (Black is unknown, Red is off, Green is on)
* Outputs have a toggle switch that can be used to change the value at the output
* GPIO configurations can be loaded at startup with a command line filename option, or loaded via file-picker from the UI or saves to file via file picker.

## New Contributors
* @andrewdavidmackenzie made their first contribution in https://github.com/andrewdavidmackenzie/pigg/pull/11
* @sundaram123krishnan made their first contribution in https://github.com/andrewdavidmackenzie/pigg/pull/12
* @dependabot made their first contribution in https://github.com/andrewdavidmackenzie/pigg/pull/24

**Full Changelog**: https://github.com/andrewdavidmackenzie/pigg/commits/0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/andrewdavidmackenzie/pigg/releases/tag/0.1.0'>v0.1.0</a>.</em>

**Comment by andrewdavidmackenzie** [comment]
Yeh! Congrats @sundaram123krishnan 

Now we can rest a little, then start scanning the list of issues and propose some for the next release by adding the "next-release" label to them!

**Reply by sundaram123krishnan** [reply]
Thx a lot @andrewdavidmackenzie

**Comment by andrewdavidmackenzie** [comment]
V0.2.0 is out, so any feedback is bestade on that thread.
